python/lsst/ts/xml/generate_subsystems_doc.py

```diff
@@ -294,9 +294,15 @@ def generate_subsystems_doc():
                         state_name = states[0].split("_")[0].lstrip()
                         cf.write(f":{state_name}:\n")
                         for state in states:
-                            cf.write(
-                                f"  * :any:`{state.split('_')[1]} <lsst.ts.idl.enums.{subsystem}.{state_name}.{state.split('_')[1].upper()}>`\n"  # noqa
-                            )
+                            # enum item format `\n enumClassName_itemName
+                            # [=value]`
+                            # where itemName may include underscores,
+                            # and all whitespace is optional and of
+                            # arbitrary length.
+                            state = state.split("_", 1)[
+                                1
+                            ]  # only split after the first underscore.
```

Comment:
You split *at* the first underscore. I think what you are trying to say is "strip the beginning through the first underscore"